main.go

```diff
@@ -68,12 +86,30 @@ func (hc *HealthCheckWebService) apiGetIsPrimary(w http.ResponseWriter, r *http.
 }
 
 func (hc *HealthCheckWebService) apiGetIsReplica(w http.ResponseWriter, r *http.Request) {
```

Comment:
because these functions do almost the same thing, should they be merged into a single function with a isPrimary flag?

Reply:
yeah, great point!

Reply:
talked with @film42 about consolidating `apiGetIsReplica` and `apiGetIsPrimary`, and we decided to handle that in a future PR